docs/api/index.rst

```diff
@@ -11,7 +11,16 @@ API
 Changelog
 ---------
 
-1.16 (unreleased)
+1.17 (unreleased)
+'''''''''''''''''
+
+- Filtering with like can now contain wild chars (eg. ``?like_nobody=*you*``)
+  It is thus now impossible to search for the ``*`` character with this operator.
+- Handle querystring parameters as JSON encoded values
+  to avoid treating number as number where they should be strings. (#1217)
+- Introduce ``has_`` filter operator (fixes #344).
```

Comment:
Oops, I didn't realize we had released a 1.16. You should probably remove the `has` and JSON-encoded notes from 1.16 (below).